src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -231,6 +237,19 @@ public static NodeSettings Default(Network network = null, ProtocolVersion proto
             nodeSettings.UseCheckpoints = config.GetOrDefault<bool>("checkpoints", true);
             nodeSettings.Logger.LogDebug("Checkpoints are {0}.", nodeSettings.UseCheckpoints ? "enabled" : "disabled");
 
+            // TODO: Figure out where to put defaults so they are easy to configure.
+            uint256 defaultAssumeValid = null;
+            if (!nodeSettings.Network.IsBitcoin() && nodeSettings.Network.IsTest())
+            {
+                // Block Height 184096 https://chainz.cryptoid.info/strat-test/block.dws?184096.htm
+                defaultAssumeValid = new uint256("0x74427b2f85b5d9658ee81f7e73526441311122f2b23702b794be557ba43ca43e");
+            }
+            // TODO: Add defaults for bitcoin and mainnet networks.
```

Comment:
These default settings should be placed somewhere that is easy to update, as you will will want to update them with each release. Network seems like logical place to me? Any better idea?

Reply:
hmm yeah the Network is probably the best candidate.
Alternatively we could put it tin the deamon.